Tasks/AzurePowerShell/AzurePowerShell.ps1

```diff
@@ -46,8 +46,8 @@ $global:ErrorActionPreference = 'Continue'
 ([scriptblock]::Create($scriptCommand)) |
     ForEach-Object {
         Remove-Variable -Name scriptCommand
-        . $_
-    } 2>&1 |
+        . $_ 2>&1
```

Comment:
I'm not sure why I had tacked on the redirection operator after the Process block parameter of the ForEach-Object command. I think it ended up there by mistake as part of last minute cleanup changes to remove local variables before calling the user's ad-hoc script.